website/sidebars.js

```diff
@@ -167,10 +281,61 @@ module.exports = {
         },
       ],
     },
+
+    // API reference
+    {
+      type: "link",
+      label: "All Providers (WIP)",
+      href: "https://github.com/rrousselGit/riverpod/tree/master/examples/marvel",
+    },
     {
       type: "link",
       label: "API reference",
       href: "https://pub.dev/documentation/hooks_riverpod/latest/hooks_riverpod/hooks_riverpod-library.html",
     },
+
+    // {
```

Comment:
Could you remove this commented code?

Reply:
This branch is still work in progress. They will be re-added in the sidebar later